metricbeat/module/zookeeper/mntr/data.go

```diff
@@ -53,7 +53,8 @@ var (
 		"approximate_data_size": c.Int("zk_approximate_data_size"),
 	}
 	schemaLeader = s.Schema{
-		"followers":        c.Int("zk_followers"),
+		"learners":         c.Int("zk_learners", s.Optional),
```

Comment:
This new `learners` field needs to be defined in `metricbeat/module/zookeeper/mntr/_meta/fields.yml`.

Reply:
> I've edited the description of the legacy field _followers_, is that OK?

Perfect, yes.

CI is failing because some files need to be updated after the change in fields.
Please run `make update` from the `metricbeat` and `x-pack/metricbeat` directories and commit the changes.

Reply:
It is weir because it still  fails after doing so, going to force a new build.